node_modules/helpers/modules/disambiguate/index.js

```diff
@@ -20,7 +20,23 @@ function userId(input, guild) {
     : queryGuildForUser(input, guild);
 }
 
+function logTypeArgument(logType) {
+    if(!logType) return 'Log';
+
+    switch (logType.toUpperCase()) {
```

Comment:
Switch statements will very quickly get ugly and difficult to manage; and also won't scale as well as you might think for the current use-case.

For this implementation I think a better approach is to create an object containing key/value pairs of your switch cases; so something like: `{"S":"Successful operation", "W":"Warning", etc...}` which we can then directly refer to by using the `in`[^1] operator:
```js
let myLogTypes = {"S":"Successful operation", "W":"Warning", etc...};
let myLogType = logType.toUpperCase();

if (myLogType in myLogTypes) return myLogTypes[myLogType];
```
If you'd like; we can move one step further and also export the object containing the log types to its own configuration file. Because this would be a configuration dependency we would have to create an additional directory in the module's root named `config` (would be located at `/logger/config/types.json` for example).

Let me know your thoughts on this.
[^1]: ['in' - JavaScript MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in)